omod/src/main/java/org/openmrs/module/smsreminder/webservice/Teste.java

```diff
@@ -0,0 +1,30 @@
+package org.openmrs.module.smsreminder.webservice;
+
+import pt.usendit.api.v2.ArrayOfSms;
+import pt.usendit.api.v2.RemoteUsendit;
+import pt.usendit.api.v2.RemoteUsenditSoap;
+import pt.usendit.api.v2.ScheduleResult;
+import pt.usendit.api.v2.Sms;
+
+public class Teste {
```

Comment:
Esta é uma classe de testes? Se for, mover para pasta de testes, se nao for Renomear para nome mais Semantico